### DISCUSSION TITLE
Compatibilidad con Contpaq 14.2.5

### DISCUSSION THREAD

**Discussion by vicman1200** [AndresRamos/Contpaqi#52] [General]
Buen día, 👋

Los SDK publicados en este respositrio son compatibles con el Contpaq 14.2.5 ?

mi principal intención es la creación de pólizas.

Gracias

**Comment by AndresRamos** [comment]
Este repositorio es para trabajar con el sistema de CONTPAQi Comercial Premium.
Para trabajar con el sistema de CONTPAQi Contabilidad deberas hacer uso del SDK de CONTPAQi Contabilidad.
El repositorio para Contabilidad es https://github.com/AndresRamos/ARSoftware.Contpaqi.Contabilidad  pero todavia no agrego codigo de SDK. Revisa en dos semanas y ya debo tener proyectos y ejemplos.
